components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -1,5 +1,11 @@
 +++
 title = "IAM Overview"
+aliases = [
+    "/docs/authorization-overview/",
+    "/docs/default-policies/",
+    "/docs/iam-v1-policies/",
+    "/docs/iam-v1-overview/"
+]
```

Comment:
maybe with this? https://gohugo.io/content-management/urls/#customize
ill prolly leave that for the docs team to decide in a later pr though

Reply:
Hm sounds promising... but I'm not fluent enough in Hugo... :/

Reply:
Not a blocking comment of course ;)

Reply:
Ok, couldn't help trying. With an `layout/alias.html` of

```
Warning: redirected because of reason

{{ .Page.Content }}
```

I see _something_ on `/iam-v1-overview`: 

![image](https://user-images.githubusercontent.com/870638/81063472-226efa80-8ed8-11ea-9906-2dab7ad575ac.png)

...not quite there yet. I'm giving up, I'm missing too much of how this works.

Also, I've noticed that there's other aliases, so we couldn't hardcode a reason there.

No idea if that's even wanted, so, dropping the case ;)